pkg/autoscaling/calculation.go

```diff
@@ -41,6 +45,12 @@ var (
 	MaxScaleInStep uint64 = 1
 )
 
+func setInformer(rc *cluster.RaftCluster) {
+	if informer == nil {
+		informer = newTidbInformer(rc.GetEtcdClient())
+	}
```

Comment:
when multiple requests arrive at the same time, is it acceptable to create multiple informer instances here?

Reply:
good catch, updated.